app/routes/contracted-hours.js

```diff
@@ -36,11 +43,34 @@ module.exports = function (router) {
       return res.sendStatus(404)
     }
 
+    try {
+      if (updatedHours) isValid(updatedHours, next)
```

Comment:
My understanding here is: if updatedHours is undefined, or otherwise falsey, the validation will not fire and there will be an attempt at writing it to the database.